mk/spksrc.service.installer

```diff
@@ -164,7 +164,7 @@ set_syno_permissions ()
             # Then fix the Synology permissions
             echo "Granting '${GROUP}' group permissions on ${DIRNAME}" >> ${INST_LOG}
             synoacltool -add "${DIRNAME}" "group:${GROUP}:allow:rwxpdDaARWcC-:fd--" >> ${INST_LOG} 2>&1
-            find "${DIRNAME}" -type d -exec synoacltool -enforce-inherit "{}" \; >> ${INST_LOG} 2>&1
+            find "${DIRNAME}" -type d -mindepth 1 -exec synoacltool -enforce-inherit "{}" \; >> ${INST_LOG} 2>&1
```

Comment:
Ok so here the issue I faced is that this command was basically ruining everything which was done before by enforcing that the rights should be inherited.

Don't ask me why but this command was basically starting by saying "on the specified folder apply inheritance rules" ...